xbmc/storage/discs/IDiscDriveHandler.h

```diff
@@ -0,0 +1,82 @@
+/*
+ *  Copyright (C) 2022 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+#pragma once
+
+#include <memory>
+#include <string>
+
+/*! \brief Represents the state of a disc (optical) drive
+*/
+enum class DriveState : unsigned int
```

Comment:
Is there a specific reason that requires `unsigned int`? 

Otherwise it is probably better to use default int to avoid possible signed/unsigned mismatch warnings in the future.

Reply:
Leftover from the last commit, those values used to (in the first commits) to be mapped to the libcdio return codes. Then I realized the actual implementation did not use them anyway. Will remove as they are not needed anymore